api_specs/v3/component_analyses_v3.yaml

```diff
@@ -0,0 +1,382 @@
+        required: false
+        schema:
+          type: string
+
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          headers:
+            uuid:
+              schema:
+                type: string
+              description: User uuid.
+          content:
+            application/json:
+              schema:
+                items:
+                  oneOf:
+                    - $ref: '#/components/schemas/FreeUser'
+                    - $ref: '#/components/schemas/NoVulnerabilities'
+              examples:
+                FreeUser:
+                  $ref: '#/components/examples/free-user'
+                No Vulnerability:
+                  $ref: '#/components/examples/no-vul'
+
+        202:
+          description: Package is currently Unavailable. Please try again later
+          content: {}
+        401:
+          description: Request unauthorized
+          content: {}
+        404:
+          description: Data not found
+          content: {}
+
+components:
+  examples:
+    free-user-batch:
+      value:
+        - package: "django"
+          version: "1.1"
+          package_unknown: false
+          vulnerabilities:
+            - id: "SNYK-2010-3082"
+              severity: "medium"
+              title: "Information Exposure"
+              url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+              fixed_in:
+                - "1.1.3"
+                - "1.2.4"
+            - id: "SYNK-2018-1002"
+              severity: "medium"
+              title: "Information Exposure"
+              url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+              fixed_in:
+                - "1.1.3"
+                - "1.2.4"
+
+        - package: "mango"
+          version: "2.1"
+          package_unknown: false
+          vulnerabilities:
+            - id: "SNYK-2010-3082"
+              severity: "medium"
+              title: "Information Exposure"
+              url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+              fixed_in:
+                - "1.1.3"
+                - "1.2.4"
+            - id: "SNYK-2010-3082"
+              severity: "medium"
+              title: "Information Exposure"
+              url: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+              fixed_in:
+                - "1.1.3"
+                - "1.2.4"
+
+    no-vul-batch:
+      value:
+        - package: "django"
+          version: "1.1"
+          recommendation: {}
+
+    free-user:
+      value:
+        package: "mango"
+        version: "2.1"
+        recommendation:
+          component_analyses:
+            vulnerability:
+              - vendor_cve_ids: "SNYK-2018-1002"
+              - vendor_cve_ids: "SNYK-2010-3082"
+          severity: "high"
+
+    no-vul:
+      value:
+        package: "mango"
+        version: "2.1"
+        recommendation: "No vulnerabilities found for package"
+
+    all-packages-unknown:
```

Comment:
@rafiu007 where will this be used?

Reply:
I had removed this .Probably got added by mistake when i copied spec from swagger hub.